testng.xml

```diff
@@ -5,24 +5,20 @@
 	<test name="TestHeadless" thread-count="2" parallel="classes">
```

Comment:
Thread Count: The thread-count="2" is fine for parallel execution but consider increasing it for large-scale tests if infrastructure supports it.
Parallelization: Classes are being run in parallel. Ensure no shared resources are accessed between tests to avoid flaky results.